recipe/meta.yaml

```diff
@@ -53,8 +54,6 @@ requirements:
 test:
   imports:
     - parcels
-    - parcels.scripts
-    - parcels.tools
```

Comment:
Looks like `parcels.scripts.get_examples`  was removed but it is still imported when importing these.

Reply:
Nope. It is the entrypoint that needs updating.

Reply:
I'm on it; will fix this today